docs/src/main/paradox/docs/getting-started/running-nexus/index.md

```diff
@@ -204,375 +189,43 @@ The provided reverse proxy (the `nginx` image) exposes several endpoints:
 If you'd like to customize the listening port or remove unnecessary endpoints, you can simply modify the `nginx.conf`
 file.
 
-## Run Nexus locally with Minikube
-
-@link:[Minikube](https://github.com/kubernetes/minikube){ open=new } is a tool that makes it easy to run Kubernetes locally. 
-Minikube runs a single-node Kubernetes cluster inside a VM on your machine for users looking to try out Kubernetes or 
-develop with it day-to-day.
-
-@@@ note
-
-This section makes use of static assets hosted on this website; to remove the clutter please export the base of the
-documentation to `$MINI` env var:
-
-```
-export MINI="https://bluebrainnexus.io/docs/getting-started/running-nexus/minikube"
-```
-
-@@@
-
-@@@ note
-
-This page presents the necessary commands to deploy Nexus with Minikube but also examples the show the expected output.
-
-Some of the examples on this page make use of @link:[curl](https://curl.se/){ open=new } and @link:[jq](https://stedolan.github.io/jq/){ open=new }
-for formatting the json output when interacting with the services. Please install these command line tools if you'd like
-to run the commands in the examples. They should be available through your usual package manager (Chocolatey, Homebrew,
-APT, YUM/DNF, ...)
-
-@@@
-
-### Minikube set-up
-
-#### Installation
-
-Follow the @link:[installation instructions](https://minikube.sigs.k8s.io/docs/start/){ open=new } from 
-the official Kubernetes documentation.
-
-#### Run Minikube
-
-To start Minikube run (notice the _cpu_ and _memory_ flags, the setup requires a minimum of `--cpus=2 --memory=8196`):
-
-```
-minikube start --cpus 6 --memory 10240 --vm-driver=$DRIVER
-```
-
-For better performance we recommended to select the `$DRIVER` [corresponding to your OS native hypervisor](https://minikube.sigs.k8s.io/docs/drivers/),
-namely _docker_ on macOS, _hyperv_ or _docker_ on Windows and _docker_ or _kvm2_ on Linux.
+### PostgreSQL partitioning 
 
-If the installation is successful you can run the following command to open the
-@link:[Kubernetes Dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/){ open=new }:
+Nexus Delta takes advantage of PostgreSQL's @link:[Table Partitioning](https://www.postgresql.org/docs/current/ddl-partitioning.html) feature. This allows for improved query performance, as well facilitates loading, deleting, or transferring data.
```

Comment:
`as well facilitates` -> `as well as facilitates` ? It seems like some word is missing with only `as well facilitates` 🤔